decentraland/screens/helpers.js

```diff
@@ -55,6 +62,40 @@ export const getContracts = () => {
   return contracts;
 };
 
+// Note: the value should be a string
+const _storeData = async (key, value) => {
+  try {
+    await SecureStore.setItemAsync(key, value);
+  } catch (error) {
+    showError("Unable to save data on storage.");
+  }
+};
+
+const _retrieveData = async key => {
+  try {
+    const value = await SecureStore.getItemAsync(key);
+    return value;
```

Comment:
`A promise that resolves to the previously stored value, or null if there is no entry for the given key. The promise will reject if an error occurred while retrieving the value.` Source: https://docs.expo.io/versions/latest/sdk/securestore/#returns-1

Should we check if value is `null` and log anything special, or is that outside of the scope of this function and something that should happen in "userland"?

Reply:
Yeah, I think it's a "userland" matter since `null` can be an OK return.

Reply:
Sounds good. Resolving this!